yelp_package/extra_requirements_yelp.txt

```diff
@@ -24,7 +24,7 @@ pyopenssl==19.0.0
 PySubnetTree==0.34
 python-jsonschema-objects==0.3.1
 retrying==1.3.3
-scribereader==0.10.0
+scribereader==0.15.0
```

Comment:
we'll also want to bump python-dateutil>=2.8.1 in requirements.txt (seems like that lower bound was added to the scribereader setup.py about a year ago)